bin/cli.js

```diff
@@ -44,6 +48,20 @@ var scope = argv.scope
 var modules = argv._
 var dir = path.resolve(process.cwd(),argv.dir)
 
+var ignorePaths = [];
+if (argv.ignorePaths.length !== 0){
+  ignorePaths = argv.ignorePaths.split(",");
+  for (var i = 0; i < ignorePaths.length; i++){
+    if (ignorePaths[i].length){
+      if (ignorePaths[i].charAt(0) !== '/'){
+        ignorePaths[i] = dir + '/' + ignorePaths[i];
```

Comment:
i prefer `path.join`  instead of `+'/'+` to be cross platform compatible. ill update this after merge.